Userland/Libraries/LibWeb/HTML/Navigator.h

```diff
@@ -65,6 +67,9 @@ class Navigator : public Bindings::PlatformObject
 
     // https://w3c.github.io/clipboard-apis/#dom-navigator-clipboard
     JS::GCPtr<Clipboard::Clipboard> m_clipboard;
+
+    // https://html.spec.whatwg.org/multipage/interaction.html#dom-navigator-useractivation
+    JS::GCPtr<UserActivation> m_user_activation;
```

Comment:
This needs to be visited in `Navigator::visit_edges()`.

Reply:
Silly me - I remember thinking I must visit this when adding to the header, but never did so. Thank you :)

I've made this change.